sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>is it possible to run this data preparation step as a lambda step instead of a processing step? We can create a zipped lambda package and Lambda step can take care of creating lambda function using that zipped code provided. Since it's output will be used in training step, we will also be able to show how to reference the outputs. It could make a good example showing all the functionalities that we provide like using a lambda with dependencies in lambda step and referencing output parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Will address in v2 of this example